test/functional/interface_usdt_net.py

```diff
@@ -142,12 +139,13 @@ def check_p2p_message(event, inbound):
                     checked_outbound_version_msg += 1
 
         def handle_inbound(_, data, __):
+            nonlocal events
```

Comment:
nit: Since we're mutating `events` (and not reassigning), there's no need to declare `nonlocal` so I think this line can be safely removed. Or, if you prefer the explicitness of altering an object from a different scope, I'd at least stay consistent and do the same in `handle_outbound`?